data/RileyMaze/panel.ui

```diff
@@ -0,0 +1,283 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated with glade 3.22.1 -->
+<interface>
+  <requires lib="gtk+" version="3.22"/>
+  <requires lib="hack-toolbox" version="0.0"/>
+  <template class="RMZCombinedTopic" parent="GtkGrid">
+    <property name="visible">True</property>
+    <property name="can_focus">False</property>
+    <property name="border_width">20</property>
+    <property name="column_spacing">20</property>
+    <child>
+      <object class="GtkGrid" id="units">
+        <property name="visible">True</property>
+        <property name="can_focus">False</property>
+        <property name="row_spacing">16</property>
+        <child>
+          <object class="Section">
```

Comment:
Border width didn't quite do what I was expecting, sorry for the wrong suggestion. I should have explained what I was aiming for in the first place instead of suggesting a solution.

In Lightspeed, there's a certain amount of space between the border of the Section widget and the widgets inside it:
![Screenshot from 2019-04-23 17-53-43](https://user-images.githubusercontent.com/459371/56625305-8e463400-65f1-11e9-8098-de08dfafacbc.png)
In the robot Section, there are only a few pixels of space between the widgets and the border:
![Screenshot from 2019-04-23 17-54-04](https://user-images.githubusercontent.com/459371/56625323-9dc57d00-65f1-11e9-9c55-73593eabb813.png)
(and now that I look closer, it looks like the dark background behind "Robot A" doesn't extend all the way to the right side of the background box.)

I guess border width is not the right way to achieve that open spacing, but maybe take a look at Lightspeed to see what's different between these two widgets?